WooCommerce/Classes/Authentication/AuthenticationManager.swift

```diff
@@ -263,36 +263,36 @@ extension AuthenticationManager: WordPressAuthenticatorDelegate {
     /// Presents the Login Epilogue, in the specified NavigationController.
     ///
     func presentLoginEpilogue(in navigationController: UINavigationController, for credentials: AuthenticatorCredentials, onDismiss: @escaping () -> Void) {
-        let matcher = ULAccountMatcher()
-        matcher.refreshStoredSites()
 
-        /// Jetpack is required. Present an error if we don't detect a valid installation for a self-hosted site.
-        if let urlFromCredentials = credentials.wpcom?.siteURL ?? credentials.wporg?.siteURL,
-           isJetpackValidForSelfHostedSite(url: urlFromCredentials) {
-            return presentJetpackError(for: urlFromCredentials, with: credentials, in: navigationController, onDismiss: onDismiss)
+        guard let siteURL = credentials.wpcom?.siteURL ?? credentials.wporg?.siteURL else {
+            DDLogError("⛔️ No site URL found to present Login Epilogue")
+            return
```

Comment:
Since `siteURL` is empty string by default, this `guard` only fails when the `credentials` contain neither `wpcom` or `wporg` info. I updated the error message in 0c8e829f4afe08ae43677abc35590f04d05ca00f.